src/HomogeneousCalorimeter_geo.cpp

```diff
@@ -1,22 +1,5 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
```

Comment:
No one else is using this, right?

Reply:
This file? No other detector, currently. Possibly, in the future this would need to, again, support some beamtest-like configurations.